resize-observer-1/Overview.bs

```diff
@@ -264,15 +279,18 @@ interface ResizeObserverEntry {
         4. Set |this|.{{ResizeObserverEntry/contentBoxSize}} slot to result of <a href="#calculate-box-size">
             computing size given |target| and observedBox of "content-box"</a>.
 
-        5. Set |this|.{{ResizeObserverEntry/contentRect}} to logical |this|.{{ResizeObserverEntry/contentBoxSize}}.
+        5. Set |this|.{{ResizeObserverEntry/devicePixelContentBoxSize}} slot to result of <a href="#calculate-box-size">
+            computing size given |target| and observedBox of "device-pixel-content-box"</a>.
 
-        6. If |target| is not an SVG element do these steps:
+        6. Set |this|.{{ResizeObserverEntry/contentRect}} to logical |this|.{{ResizeObserverEntry/contentBoxSize}}.
```

Comment:
"... given |target| and and observedBox of content-box" ?

Reply:
done, thanks!